picard/ui/mainwindow.py

```diff
@@ -104,13 +105,16 @@ class MainWindow(QtWidgets.QMainWindow, PreserveGeometry):
         config.BoolOption("persist", "view_toolbar", True),
         config.BoolOption("persist", "view_file_browser", False),
         config.TextOption("persist", "current_directory", ""),
+        config.IntOption("persist", "mediaplayer_volume", "50"),
     ]
 
     def __init__(self, parent=None):
         super().__init__(parent)
         self.selected_objects = []
         self.ignore_selection_changes = False
         self.toolbar = None
+        self.player = QtMultimedia.QMediaPlayer(self)
```

Comment:
IMHO, we should test availability using https://doc.qt.io/qt-5/qmediaplayer.html#availability and check if it is `QMultimedia.Available`.
If not available, all associated code should be skipped (and buttons disabled / hidden)

Reply:
@zas, I thought that it should be in the next pull request (with checking if the QtMultimedia module available).

Reply:
Ok